docs/src/_reference/settings.md

```diff
@@ -254,6 +254,28 @@ meltano config meltano set hub_url http://localhost:4000
 export MELTANO_HUB_URL=http://localhost:4000
```

Comment:
@DanielPDWalker can you update the env var and `meltano config meltano set hub_url ...` examples to reflect the new expectation?

```suggestion
export MELTANO_HUB_URL=http://localhost:4000/meltano/api/v1
```

Reply:
Sure thing, Ill make the changes tomorrow.